pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentDispatcherSingleActiveConsumer.java

```diff
@@ -293,6 +293,12 @@ public void redeliverUnacknowledgedMessages(Consumer consumer, long consumerEpoc
     }
 
     private synchronized void internalRedeliverUnacknowledgedMessages(Consumer consumer, long consumerEpoch) {
+
+        if (consumerEpoch > consumer.getConsumerEpoch()) {
+            log.info("set epoch: " + consumerEpoch);
```

Comment:
Should be a debug level log, otherwise, for each redelivery request, the broker will print log here.
And it's better to provide more information in the log, such as the topic name, the subscription name, the consumer id/name

Reply:
Thanks, fixed.